ci/travis-27.yaml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - cython=0.28.2
   - fastparquet
   - feather-format
+  - futures
```

Comment:
is this just for testing?

Reply:
Yes, in the test file `pandas/tests/indexes/test_base.py` we copied the code from the issue reported in GitHub that imports a `ThreadPoolExecutor`, available in Python 2.7 only with `futures` package. 

```python
from concurrent.futures import ThreadPoolExecutor
```

Reply:
This will I think cause failures for people who just install pandas and run `pandas.test`.

I'd do that in a pytest `importorskip`.